--- origsrc/unique-1.0.6/unique/dbus/Makefile.am	2008-07-31 08:26:32.000000000 -0500
+++ src/unique-1.0.6/unique/dbus/Makefile.am	2009-02-11 20:26:56.250000000 -0600
@@ -25,12 +25,12 @@ uniquebackend-bindings.h: uniquebackend-
 		--mode=glib-server \
 		--output=$@ \
 		--prefix=uniquebackend \
-	uniquebackend-dbus.xml
+	$<
 uniquebackend-glue.h: uniquebackend-dbus.xml $(DBUS_BINDING_TOOL)
 	$(DBUS_BINDING_TOOL) \
 		--mode=glib-client \
 		--output=$@ \
 		--prefix=uniquebackend \
-	uniquebackend-dbus.xml
+	$<
 
 EXTRA_DIST = uniquebackend-dbus.xml
